game_patch/rf/multi.h

```diff
@@ -151,6 +165,10 @@ namespace rf
     static auto& multi_tdm_get_red_team_score = addr_as_ref<int()>(0x004828F0); // returns ubyte in vanilla game
     static auto& multi_tdm_get_blue_team_score = addr_as_ref<int()>(0x00482900); // returns ubyte in vanilla game
     static auto& multi_num_players = addr_as_ref<int()>(0x00484830);
+    [[nodiscard]] inline int32_t multi_num_spawned_players() {
+        const auto f = [] (auto& p) { return !player_is_dead(&p) && !player_is_dying(&p); };
```

Comment:
`f` is not very descriptive, maybe `is_alive`?

Reply:
I did not want to split this line. It works better in a single line imo.